styleguide/css/main.scss

```diff
@@ -47,9 +43,50 @@ $medium-gray: #666;
 }
 
 .openbtn {
-  background-color: $dark-gray;
+  background-color: $gray-dark;
   left: 0;
   padding: 10px;
   position: absolute;
   top: 0;
 }
+
+.anchored-text {
+  @include border-radius(1.5rem);
+  left: 0;
+  padding: 1.5rem;
+  position: absolute;
+  text-indent: -100000px;
+  top: 0;
+  z-index: 99999;
```

Comment:
yeah I just through some stuff together.  we can adjust the styling.  no biggie.